src/raftstore/store/fsm/peer.rs

```diff
@@ -1047,7 +1048,7 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         // is the state of target peer at the time when source peer is merged. So here we record the
         // merge target epoch version to let the target peer on this store to decide whether to
         // destroy the source peer.
-        let mut meta = self.ctx.store_meta.lock().unwrap();
+        let mut meta = self.ctx.store_meta.inner.lock().unwrap();
```

Comment:
seem we can use HandyRwLock to simplify the code.

Reply:
It's `Mutex`, not `RwLock`.